architecture/acs-architecture.adoc

```diff
@@ -9,10 +9,17 @@ Discover {product-title} architecture and concepts.
 
 include::modules/acs-architecture-overview.adoc[leveloffset=+1]
 
-include::modules/acs-architecture-external-components.adoc[leveloffset=+1]
-
 [role="_additional-resources"]
 .Additional resources
-* xref:../integration/integrate-with-image-registries.adoc[Integrating with image registries]
-* xref:../integration/integrate-with-ci-systems.adoc[Integrating with CI systems]
-* xref:../integration/integrate-using-syslog-protocol.adoc[Integrating by using the syslog protocol]
+* xref:../architecture/acs-architecture.adoc#acs-architecture-diffrences-ocp-kube_acs-architecture[Architectural differences between installation on {ocp} and Kubernetes]
+* xref:../architecture/acs-architecture.adoc#external-components_acs-architecture[External components]
+
+include::modules/acs-central-services-overview.adoc[leveloffset=+1]
+
+include::modules/acs-secured-cluster-services-overview.adoc[leveloffset=+1]
+
+include::modules/acs-architecture-external-components.adoc[leveloffset=+1]
+
+include::modules/acs-architecture-diffrences-ocp-kube.adoc[leveloffset=+1]
+
+include::modules/acs-architecture-interaction-between-services.adoc[leveloffset=+1]
```

Comment:
While not directly related tot his PR, please update the meta data of this file to include `:content-type:`